extensions/codeception/TestCase.php

```diff
@@ -126,6 +126,9 @@ protected function mockApplication($config = null)
      */
     protected function destroyApplication()
     {
+        if (Yii::$app->get('session', false)) {
+            Yii::$app->session->destroy();
+        }
         Yii::$app = null;
```

Comment:
May be. But this code will destroy a session and it will prevent developer from nullifying the app prior to session destroy.
Such behavior guarantees absence of the bug that this fix relates to.